src/main/java/com/panasi/interview_questions/repository/dto/QuestionDto.java

```diff
@@ -9,7 +9,8 @@ public class QuestionDto {
 	
 	private Integer id;
 	private String name;
+	private Integer categoryId;
 	private List<AnswerDto> answers;
-	private CategoryDto category;
```

Comment:
but here better to use Category, to show on page this category (categoryId can be anough, it can depend on frontend implementation)

Reply:
I decided that it is easier to specify the category id when creating a question